src/Core/src/Platform/iOS/MauiTextView.cs

```diff
@@ -127,7 +127,8 @@ MauiLabel InitPlaceholderLabel()
 			{
 				BackgroundColor = UIColor.Clear,
 				TextColor = ColorExtensions.PlaceholderColor,
-				Lines = 0
+				Lines = 0,
+				VerticalAlignment = UIControlContentVerticalAlignment.Top
```

Comment:
`_placeholderLabel ` VerticalAlignment is Center default. The `UpdatePlaceholderLabelFrame `method update the placeholder UILabel Frame (position) taking into account the size and the UIEdgeInsets.